README.md

```diff
@@ -16,6 +16,25 @@ Elemental is a tool for installing, configuring and updating operating system im
 *   **Updates:** Update an existing OS installation from a newer image.
 *   **Extensibility:** Extend the OS installation image with extensions.
 
+## Quick-start
+
+Elemental can be used to convert an OCI image into a bootable disk-image. This requires that the OCI image actually contains a bootloader, kernel, initrd and init-system. In this quick-start we will use the [tumbleweed example](./examples/tumbleweed/Dockerfile).
```

Comment:
thank you so much for extending the documentation. jfyi, I've been populating the open build service with a tumbleweed based image:

`podman pull registry.opensuse.org/devel/unifiedcore/tumbleweed/containers/uc-base-os-kernel-default:0.0.1`

so we could use that instead of copying the dockerfile in verbatim into the repository also. not sure what's better (having it in three or pre-built for people start right away with)

Reply:
I have no strong opinion either, having a reference image pre-built somewhere else is nice and simplifies the repository and required steps. IMHO, if we are not considering embracing derived images as an strategy for customization I'd probably not include the Dockerfile here and simply refer to that public image Dirk refers to.